extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -97,7 +98,8 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
    * apply when data is parsed.
    */
   fun data(value: String) = apply {
-    ignoreParsedGeoJsonRegistry[sourceId] = true
+    // Clear all asynchronous data
+    updateParsedGeoJsonRegistry.clear()
```

Comment:
shouldn't actually it be the whole fix?
why do we need a set instead of the hash map?

Reply:
The set contains those data that need to be updated to source, no need additional true or false values for them. In other words, those data that are not in the set will not be updated to the source